bin/custom/concerttours/resources/concerttours-items.xml

```diff
@@ -117,6 +122,10 @@
                     <description>type of concert (indoors or open air)</description>
                     <persistence type="property"/>
                 </attribute>
+                <attribute qualifier="daysUntil" type="java.lang.Long">
```

Comment:
naming of an attribute is not clear: days until what? Isn't it better to rename to smth like daysUntillConcertLeft? + including a handler